mesh/mesh.go

```diff
@@ -424,7 +424,7 @@ func (m *Mesh) AccumulateRewards(rewardLayer types.LayerID, params Config) {
 	for _, bl := range l.Blocks() {
 		atx, err := m.AtxDB.GetAtx(bl.ATXID)
 		if err != nil {
-			m.Error("Atx not found %v block %v", err, bl.Id)
+			m.Warning("Atx not found %v block %v", err, bl.Id)
```

Comment:
1. Use `.With()`
2. This should only be a `Warning` if `err == database.ErrNotFound`, otherwise it should stay an `Errror` (maybe with a different message)

Reply:
Fixed